easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.7.1'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
```

Comment:
@branfosj Can you change this to `2021.04` (the temporary version for what is likely to become `gompi/2021a`)? Cfr. ~~#12571~~

Reply:
done in f5e9295